Code/src/main/java/nl/utwente/group10/haskell/expr/Ident.java

```diff
@@ -0,0 +1,42 @@
+package nl.utwente.group10.haskell.expr;
+
+import nl.utwente.group10.haskell.env.Env;
+import nl.utwente.group10.haskell.exceptions.HaskellTypeError;
+import nl.utwente.group10.haskell.hindley.GenSet;
+import nl.utwente.group10.haskell.type.Type;
+
+/**
+ * Expression contained in the environment.
+ */
+public class Ident extends Expr {
+    /**
+     * Name of this identity.
```

Comment:
Not really, `Ident` is short for *identifier* not *identity*, and `name` is a bit of a misnomer: it's really the identifier itself.